### ISSUE TITLE
Empty Project

### ISSUE THREAD

**Issue by ghost** [simonng/Dropbox-API-Demo#1]
Hi, thanks for the nice tutorial.
However, this project seems to be an empty one.
Cheers!

**Comment by rohn-patel** [comment]
Yep. Strange no-one is paying attention on github though. :/ Let me try to reach simon on twitter.

**Comment by nbhasin2** [comment]
@simonng This project seem to be empty
